src/date/format-properties.js

```diff
@@ -19,11 +19,12 @@ define([
  *
  * TODO Support other calendar types.
  */
-return function( pattern, cldr ) {
+return function( pattern, cldr, timeZone ) {
 	var properties = {
 			numberFormatters: {},
 			pattern: pattern,
-			timeSeparator: numberSymbol( "timeSeparator", cldr )
+			timeSeparator: numberSymbol( "timeSeparator", cldr ),
+			timeZone: timeZone
```

Comment:
Let's skip adding `timeZone` to the properties. We can add it later if required.